tests/test_cluster_read.py

```diff
@@ -29,6 +29,18 @@ def test_date(self):
         self.assertGreaterEqual(int(v), before)
         self.assertGreaterEqual(after, int(v))
 
+    def test_count(self):
+        string = """
+            - name: measurements.logs.openshift-pipelines.pipelines-as-code-controller
+              log_source_command: oc -n openshift-pipelines logs --since=10h --all-containers --selector app.kubernetes.io/component=controller,app.kubernetes.io/instance=default
```

Comment:
You can hack this and use something like `echo -e "log_line_1\nlog_line_2\n..."` here to avoid need to call oc here - oc was meant just as an example of how to get a log.